README.md

```diff
@@ -12,3 +12,19 @@ page](https://docs.github.com/en/get-started/writing-on-github/working-with-adva
 might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
+
+I am a little bit do not know how to write in big O format. The following are just mathematical calculations with what I thought big O.
+
+$T(\log_{5}n)  \leq O(log_{2}n)$
+
+$T(\log_{5}n)  \leq c * O(log_{2}n)$ 
+
+$if\ c = log_{5}2\$
```

Comment:
You can't assume a specific $c$ in this proof.

Reply:
Okay, I changed it.